config/init.php

```diff
@@ -1,3 +1,6 @@
 <?php
 include('config/config.php');
 include('include/db_query.php'); //this should happen right after config so other functions have access to the database
+include ('include/header.php');
+include('include/posts.php');
+date_default_timezone_set('America/Chicago');
```

Comment:
I'd suggest putting this at the very top of this file. Any error that occurs before this runs (like if one of the included files has an error) will print that annoying timezone warning if this line hasn't been called yet.